apps/www/src/app/(public pages)/hiring-managers/learn/(pages)/about-digital-talent.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: About the Digital Talent and Attraction Development Team
-description: A place for you to test out your Markdoc
+title: About the Digital Talent Attraction and Development Team
+description: British Columbia, BC Government, hiring servies, Office of the Chief Information Officer, about us, Digital Talent Attraction and Development Team, DTAD

```

Comment:
To clarify, (this)[https://www.shopify.com/ca/blog/how-to-write-meta-descriptions#1] is what I have in mind for the description -- it will show up in search engine results so perhaps an explicit statement would work better.

Reply:
T mentioned using numerals in place of the spelled out numbers, _or_, using th spelled out version and numbers in (), either before or after.